### DISCUSSION TITLE
Do I need "Podman Update" for "Multicast Relay"?

### DISCUSSION THREAD

**Discussion by VeniceNerd** [unifi-utilities/unifi-common#279] [Q&A] [answered]
Hey guys,

I have installed "on-boot-script" and "container-common" on my UPDMP because I would like to run the "multicast-relay". However, I'm wondering if I will also need to install "Podman Update" before I can go ahead with the "multicast-relay" install.

I looked at the guide for "Podman Update" and it looks rather intimidating so I'd rather skip this unless it is required to get "multicast-relay" working reliably.

Would appreciate any insights on this! :)

**Comment by boostchicken** [comment, marked as answer]
Nope you never need it on the UDMP.

**Comment by boostchicken** [comment]
To that end I don't even suggest it unless you need some of the features it offers, its more headache than value unless you really need somethings like docker-compose support or the remote daemon so you can act on containers on the UDMP from external systems.  For most use cases, avoud it.

**Comment by VeniceNerd** [comment]
You have no idea how glad I am to hear this. Hahaha. One less thing to worry about. Thank you!
